[2024-07-15T22:42:36.467+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-07-15T22:42:36.475+0000] {__init__.py:51} DEBUG - Loading core task runner: StandardTaskRunner
[2024-07-15T22:42:36.481+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-07-15T22:42:36.483+0000] {base_task_runner.py:68} DEBUG - Planning to run as the  user
[2024-07-15T22:42:36.484+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_init_tasks.dbt_seed scheduled__2024-07-14T00:00:00+00:00 [queued]> from DB
[2024-07-15T22:42:36.493+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_init_tasks.dbt_seed scheduled__2024-07-14T00:00:00+00:00 [queued]> dependency 'Task Instance State' PASSED: True, Task state queued was valid.
[2024-07-15T22:42:36.494+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_init_tasks.dbt_seed scheduled__2024-07-14T00:00:00+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-07-15T22:42:36.497+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_init_tasks.dbt_seed scheduled__2024-07-14T00:00:00+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2024-07-15T22:42:36.498+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_init_tasks.dbt_seed scheduled__2024-07-14T00:00:00+00:00 [queued]> dependency 'Task Instance Not Running' PASSED: True, Task is not in running state.
[2024-07-15T22:42:36.498+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: run_dbt_init_tasks.dbt_seed scheduled__2024-07-14T00:00:00+00:00 [queued]>
[2024-07-15T22:42:36.499+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_init_tasks.dbt_seed scheduled__2024-07-14T00:00:00+00:00 [queued]> dependency 'Task Concurrency' PASSED: True, Task concurrency is not set.
[2024-07-15T22:42:36.502+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_init_tasks.dbt_seed scheduled__2024-07-14T00:00:00+00:00 [queued]> dependency 'Pool Slots Available' PASSED: True, There are enough open slots in default_pool to execute the task
[2024-07-15T22:42:36.502+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_init_tasks.dbt_seed scheduled__2024-07-14T00:00:00+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2024-07-15T22:42:36.503+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_init_tasks.dbt_seed scheduled__2024-07-14T00:00:00+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-07-15T22:42:36.503+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: run_dbt_init_tasks.dbt_seed scheduled__2024-07-14T00:00:00+00:00 [queued]>
[2024-07-15T22:42:36.503+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2024-07-15T22:42:36.508+0000] {taskinstance.py:1327} INFO - Executing <Task(DbtSeedOperator): dbt_seed> on 2024-07-14 00:00:00+00:00
[2024-07-15T22:42:36.510+0000] {standard_task_runner.py:57} INFO - Started process 456 to run task
[2024-07-15T22:42:36.512+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'run_dbt_init_tasks', 'dbt_seed', 'scheduled__2024-07-14T00:00:00+00:00', '--job-id', '72', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt_init_tasks.py', '--cfg-path', '/tmp/tmp441oll6j']
[2024-07-15T22:42:36.512+0000] {standard_task_runner.py:85} INFO - Job 72: Subtask dbt_seed
[2024-07-15T22:42:36.512+0000] {cli_action_loggers.py:65} DEBUG - Calling callbacks: [<function default_action_log at 0xffff7f47a550>]
[2024-07-15T22:42:36.533+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-07-15T22:42:36.535+0000] {task_command.py:410} INFO - Running <TaskInstance: run_dbt_init_tasks.dbt_seed scheduled__2024-07-14T00:00:00+00:00 [running]> on host b91763c3682d
[2024-07-15T22:42:36.535+0000] {settings.py:353} DEBUG - Disposing DB connection pool (PID 456)
[2024-07-15T22:42:36.535+0000] {settings.py:212} DEBUG - Setting up DB connection pool (PID 456)
[2024-07-15T22:42:36.535+0000] {settings.py:285} DEBUG - settings.prepare_engine_args(): Using NullPool
[2024-07-15T22:42:36.536+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_init_tasks.dbt_seed scheduled__2024-07-14T00:00:00+00:00 [running]> from DB
[2024-07-15T22:42:36.551+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-07-15T22:42:36.552+0000] {taskinstance.py:868} DEBUG - Clearing XCom data
[2024-07-15T22:42:36.563+0000] {retries.py:80} DEBUG - Running RenderedTaskInstanceFields._do_delete_old_records with retries. Try 1 of 3
[2024-07-15T22:42:36.569+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='run_dbt_init_tasks' AIRFLOW_CTX_TASK_ID='dbt_seed' AIRFLOW_CTX_EXECUTION_DATE='2024-07-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-14T00:00:00+00:00'
[2024-07-15T22:42:36.569+0000] {__init__.py:117} DEBUG - Preparing lineage inlets and outlets
[2024-07-15T22:42:36.569+0000] {__init__.py:155} DEBUG - inlets: [], outlets: []
[2024-07-15T22:42:36.569+0000] {dbt_hook.py:117} INFO - /home/airflow/.local/bin/dbt seed
[2024-07-15T22:42:36.572+0000] {dbt_hook.py:126} INFO - Output:
[2024-07-15T22:42:37.583+0000] {dbt_hook.py:130} INFO - [0m22:42:37  Running with dbt=1.8.3
[2024-07-15T22:42:37.688+0000] {dbt_hook.py:130} INFO - [0m22:42:37  Registered adapter: ***=1.8.2
[2024-07-15T22:42:37.935+0000] {dbt_hook.py:130} INFO - [0m22:42:37  Found 1 model, 3 sources, 652 macros
[2024-07-15T22:42:37.937+0000] {dbt_hook.py:130} INFO - [0m22:42:37
[2024-07-15T22:42:37.939+0000] {dbt_hook.py:130} INFO - [0m22:42:37  Nothing to do. Try checking your model configs and model specification args
[2024-07-15T22:42:37.971+0000] {dbt_hook.py:130} INFO - [0m22:42:37
[2024-07-15T22:42:37.971+0000] {dbt_hook.py:130} INFO - [0m22:42:37  [32mCompleted successfully[0m
[2024-07-15T22:42:37.971+0000] {dbt_hook.py:130} INFO - [0m22:42:37
[2024-07-15T22:42:37.972+0000] {dbt_hook.py:130} INFO - [0m22:42:37  Done. PASS=0 WARN=0 ERROR=0 SKIP=0 TOTAL=0
[2024-07-15T22:42:38.449+0000] {dbt_hook.py:132} INFO - Command exited with return code 0
[2024-07-15T22:42:38.449+0000] {__init__.py:75} DEBUG - Lineage called with inlets: [], outlets: []
[2024-07-15T22:42:38.450+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_init_tasks.dbt_seed scheduled__2024-07-14T00:00:00+00:00 [running]> from DB
[2024-07-15T22:42:38.456+0000] {taskinstance.py:1356} DEBUG - Clearing next_method and next_kwargs.
[2024-07-15T22:42:38.456+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=run_dbt_init_tasks, task_id=dbt_seed, execution_date=20240714T000000, start_date=20240715T224236, end_date=20240715T224238
[2024-07-15T22:42:38.456+0000] {taskinstance.py:2305} DEBUG - Task Duration set to 1.957855
[2024-07-15T22:42:38.462+0000] {cli_action_loggers.py:83} DEBUG - Calling callbacks: []
[2024-07-15T22:42:38.487+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-07-15T22:42:38.497+0000] {dagrun.py:697} DEBUG - number of tis tasks for <DagRun run_dbt_init_tasks @ 2024-07-14 00:00:00+00:00: scheduled__2024-07-14T00:00:00+00:00, state:running, queued_at: 2024-07-15 22:42:32.293100+00:00. externally triggered: False>: 4 task(s)
[2024-07-15T22:42:38.497+0000] {dagrun.py:718} DEBUG - number of scheduleable tasks for <DagRun run_dbt_init_tasks @ 2024-07-14 00:00:00+00:00: scheduled__2024-07-14T00:00:00+00:00, state:running, queued_at: 2024-07-15 22:42:32.293100+00:00. externally triggered: False>: 3 task(s)
[2024-07-15T22:42:38.497+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_init_tasks.model.ecomm_demo.dim_date scheduled__2024-07-14T00:00:00+00:00 [None]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2024-07-15T22:42:38.497+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_init_tasks.model.ecomm_demo.dim_date scheduled__2024-07-14T00:00:00+00:00 [None]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-07-15T22:42:38.498+0000] {taskinstance.py:1103} DEBUG - Dependencies all met for dep_context=None ti=<TaskInstance: run_dbt_init_tasks.model.ecomm_demo.dim_date scheduled__2024-07-14T00:00:00+00:00 [None]>
[2024-07-15T22:42:38.498+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_init_tasks.test.ecomm_demo.dim_date scheduled__2024-07-14T00:00:00+00:00 [None]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2024-07-15T22:42:38.498+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_init_tasks.test.ecomm_demo.dim_date scheduled__2024-07-14T00:00:00+00:00 [None]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-07-15T22:42:38.498+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_init_tasks.test.ecomm_demo.dim_date scheduled__2024-07-14T00:00:00+00:00 [None]> dependency 'Trigger Rule' PASSED: False, Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_states=_UpstreamTIStates(success=1, skipped=0, failed=0, upstream_failed=0, removed=0, done=1), upstream_task_ids={'model.ecomm_demo.dim_date', 'dbt_seed'}
[2024-07-15T22:42:38.498+0000] {taskinstance.py:1093} DEBUG - Dependencies not met for <TaskInstance: run_dbt_init_tasks.test.ecomm_demo.dim_date scheduled__2024-07-14T00:00:00+00:00 [None]>, dependency 'Trigger Rule' FAILED: Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_states=_UpstreamTIStates(success=1, skipped=0, failed=0, upstream_failed=0, removed=0, done=1), upstream_task_ids={'model.ecomm_demo.dim_date', 'dbt_seed'}
[2024-07-15T22:42:38.498+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_init_tasks.generate_dbt_docs scheduled__2024-07-14T00:00:00+00:00 [None]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2024-07-15T22:42:38.499+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_init_tasks.generate_dbt_docs scheduled__2024-07-14T00:00:00+00:00 [None]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-07-15T22:42:38.499+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_init_tasks.generate_dbt_docs scheduled__2024-07-14T00:00:00+00:00 [None]> dependency 'Trigger Rule' PASSED: False, Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 2 non-success(es). upstream_states=_UpstreamTIStates(success=0, skipped=0, failed=0, upstream_failed=0, removed=0, done=0), upstream_task_ids={'model.ecomm_demo.dim_date', 'test.ecomm_demo.dim_date'}
[2024-07-15T22:42:38.499+0000] {taskinstance.py:1093} DEBUG - Dependencies not met for <TaskInstance: run_dbt_init_tasks.generate_dbt_docs scheduled__2024-07-14T00:00:00+00:00 [None]>, dependency 'Trigger Rule' FAILED: Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 2 non-success(es). upstream_states=_UpstreamTIStates(success=0, skipped=0, failed=0, upstream_failed=0, removed=0, done=0), upstream_task_ids={'model.ecomm_demo.dim_date', 'test.ecomm_demo.dim_date'}
[2024-07-15T22:42:38.502+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
